example/config/.mocharc.jsonc

```diff
@@ -10,5 +10,7 @@
   "reporter": /* 📋 */ "spec",
   "slow": 75,
   "timeout": 2000,
-  "ui": "bdd"
+  "ui": "bdd",
+  "watchFiles": ["lib/**/*.js", "test/**/*.js"],
```

Comment:
Here you could leave it as is, and add a comment, that camelcased versions do work as well.

Reply:
Fixed in https://github.com/mochajs/mocha/pull/3980/commits/f18a0b66b5df455fc7b1598797e0e3245df52826